src/theory/quantifiers/quantifiers_attributes.cpp

```diff
@@ -50,8 +50,9 @@ void QuantAttributes::setUserAttribute( const std::string& attr, Node n, std::ve
     SygusAttribute ca;
     n.setAttribute( ca, true );
   }
-  else if (attr == "quant-name")
+  else if (attr == "qid")
   {
+    // using z3 syntax "qid"
```

Comment:
I was going to ask why use this rather than `:named` but I think this comment answers the question. :)